.env.development

```diff
@@ -0,0 +1,8 @@
+module.export = {
```

Comment:
Den her og den tilsvarende for produktionen bør vist bo på -config repoet i stedet.

Reply:
Enig. Og token bør teknisk set roteres, nu det er postet til Github. 
(Selv om jeg formoder den ender på et public site under alle omstændigheder.)

Reply:
Det kunne nok være en ide at tilføje `.env.*` til `.gitignore`, så kommer de ikke med ved en fejl i fremtiden

Reply:
Får jeg gjort :) Tak for at gøre mig opmærksom på det.

Reply:
Får også kigget på at få sidens token roteret.